future-apps/payroll/contracts/Payroll.sol

```diff
@@ -32,10 +31,11 @@ contract Payroll is EtherTokenConstant, IForwarder, IsContract, AragonApp {
     bytes32 constant public MODIFY_RATE_EXPIRY_ROLE = keccak256("MODIFY_RATE_EXPIRY_ROLE");
 
     uint128 internal constant ONE = 10 ** 18; // 10^18 is considered 1 in the price feed to allow for decimal calculations
+    uint256 internal constant MAX_ALLOWED_TOKENS = 20; // for loop in `payday()` uses ~270k gas per token
+    uint256 internal constant MAX_UINT256 = uint256(-1);
```

Comment:
If you agree, I'd prefer being explicit then :)

Reply:
There is no problem on changing it at all, but I think we are losing information if we do. For example, I always try to use `uint256` instead of `uint`, but I also know that `SafeMath` has been out there for a while haha. Anyway, let's pick a convention and move forward. I'll let you both decide :)

Reply:
Will move to `UINT256` in Finance (it's also easier to search for 😄)